app/javascript/Application/util/ApiErrorHandler.js

```
@@ -17,17 +17,37 @@ function postErrors(issueDetails) {
 }
 
 export function handleError(error) {
-  const errorResponse = error.response;
-  if (errorResponse && errorResponse.status === 409) {
-    throw error;
-  }
+  processEmptyResponse(error);
+  processStatuses(error);
+  processIssueDetails(error);
+  defaultErrorHandler(error);
+}
 
-  if (errorResponse && errorResponse.data && errorResponse.data.issue_details) {
-    postErrors(errorResponse.data.issue_details);
-  } else {
+function processEmptyResponse(error) {
+  if (!error.response) {
     postError(error.message);
   }
+}
+
+function processStatuses(error) {
+  switch (error.response.status) {
```

`error.response` might be `null` or `undefined` and logic will be broken. `processEmptyResponse` should throw an error after post to resolve this issue